src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -40,6 +48,9 @@ public RobotContainer() {
   }
 
   private void setDefaultCommands() {
+
+    //arm.setDefaultCommand(new InstantCommand(() -> {arm.driveArm());
```

Comment:
uncomment this and make it take in joystick input. It should do something.
it should probably use armTeleop?